docs/configuration.md

```diff
@@ -52,6 +52,22 @@ Configuration for Atlas Checks uses a HOCON (Human Optimized Config Object Notat
  This configuration file is the default configuration file supplied with Atlas Checks and configures the all checks
  that are included with Atlas Checks.
  
+ ### Groups
+
+A configuration file can optionally have a groups object.
+The keys in this object are the names of groups, and the values are lists of ISO3 country codes that are present in those groups.
+For example, given the following groups object:
+```
+{
+  "groups": {
+    "one": ["ABC", "DEF", "GHI"],
+    "two": ["JKL", "MNO", "PQR"]
+  }
+}
+```
+the rest of the file could refer to the fictional countries with ISO codes ABC, DEF, and GHI with the string “one”, and the fictional countries with the codes JKL, MNO, and PQR with the string “two”.
+One use for this is in the override object, discussed below.
```

Comment:
Do you mind adding an example of this as well

Reply:
An example of groups in the override object starts on line 126 🙂

Reply:
duhhh